website/content/en/docs/cli/operator-sdk_run_bundle.md

```diff
@@ -29,7 +29,7 @@ operator-sdk run bundle <bundle-image> [flags]
 
 ```
       --ca-secret-name string                     Name of a generic secret containing a PEM root certificate file required to pull bundle images. This secret *must* be in the namespace that this command is configured to run in, and the file *must* be encoded under the key "cert.pem"
-      --decompression-image string                image used in an init container in the registry pod to decompress the compressed catalog contents. cat and gzip binaries are expected to exist in the PATH (default "registry.access.redhat.com/ubi8:8.9")
+      --decompression-image string                image used in an init container in the registry pod to decompress the compressed catalog contents. cat and gzip binaries are expected to exist in the PATH (default "registry.access.redhat.com/ubi9/ubi-minimal:9.4")
```

Comment:
Is this the right image to use? Does moving down from UBI to Minimal work as expected?

Reply:
copy pasta issue